lib/Plerd/Post.pm

```diff
@@ -60,6 +60,12 @@ has 'attributes' => (
     isa => 'HashRef',
 );
 
+has 'tags' => (
```

Comment:
This attribute ought to be documented in the appropriate section of the POD at the end of this file.